psi4/driver/p4util/fcidump.py

```diff
@@ -39,7 +39,7 @@
 from .exceptions import ValidationError, TestComparisonError
 
 
-def fcidump(wfn, fname='INTDUMP', oe_ints=None):
+def fcidump(wfn, fname='INTDUMP', oe_ints=None, write_pntgrp=False):
     """Save integrals to file in FCIDUMP format as defined in Comp. Phys. Commun. 54 75 (1989)
     Additional one-electron integrals, including orbital energies, can also be saved.
```

Comment:
Could you add to the docstring that the `PNTGRP` label is not in the Comp Phys Chem standard and might be incompatible with other codes?